recipes/lua-language-server/recipe.yaml

```diff
@@ -0,0 +1,45 @@
+context:
+  name: lua-language-server
+  version: 3.14.0
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  git: https://github.com/LuaLS/${{ name }}
+  tag: ${{ version }}
+
+build:
+  number: 0
+  script: build.sh
+  skip:
+    - win
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('cxx') }}
+    - ninja
+    - coreutils
+    - diffutils
+    - if: linux
+      then:
+        - gcc  # required for gcc/ar shortcuts to luamake build
```

Comment:
```suggestion
```

Reply:
The preferred solution here is to make a symlink - I'll post this in the other suggestion.

Reply:
Done at ece4115